packages/tools/webpack-fluid-loader/src/loader.ts

```diff
@@ -243,9 +244,21 @@ export async function start(
         getFluidObjectAndRender(container1, fluidObjectUrl, leftDiv).catch(() => { });
     });
 
+    let currentContainer1 = container1;
     // We have rendered the Fluid object. If the container is detached, attach it now.
-    if (container1.attachState === AttachState.Detached) {
-        await attachContainer(container1, urlResolver, documentId, url, rightDiv, manualAttach);
+    if (currentContainer1.attachState === AttachState.Detached) {
+        const result = await attachContainer(
```

Comment:
If possible, I'd remove currentContainer1 , and just assign result to container1  here

Reply:
Done.